src/app/home/home.page.ts

```diff
@@ -36,9 +36,9 @@ export class HomePage implements OnInit {
 
   ionViewWillEnter() {
     this._pictureService.init();
-    this._loadingService.show({message: "..loading.."}).then(() => {
-
-    })
+    //this._loadingService.show({message: "..loading.."}).then(() => {
```

Comment:
This particular change, these three lines, should not go in this PR. Please submit a new PR with these changes.

Reply:
I am not seeing this